test/unit/io/alignment_file/alignment_file_format_test_template.hpp

```diff
@@ -388,6 +375,23 @@ TYPED_TEST_P(alignment_file_read, format_error_ref_id_not_in_reference_informati
 // alignment_file_write
 // ----------------------------------------------------------------------------
 
+// Note that these differ from the alignment_file_output default fields:
+// 1. They don't contain field::bit_score and field::evalue since these belong to the BLAST format.
```

Comment:
This reasoning suggests that this file is already specialised for sam/bam. Can this template file still be easily included and used for BLAST (and other) format tests? If the answer is no, I suggest to rename this file into something more specific to avoid confusion in case someone implements another alignment format. It seems like the formats are so different that we cannot have a single template anyway?

Can be discussed independently of this PR.

Reply:
Yes, you are right, this test template would not work for BLAST just like this.
It would need to be split into a generic alignment format template and a sam/bam template. I would suggest doing so when BLAST will be implemented to avoid code refactorings / renamings before the new format can be tested with it?

Reply:
Yes, alright! :)